.github/workflows/linux-builds-on-master.yaml

```diff
@@ -130,6 +135,25 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         if: github.event_name == 'push' && github.ref == 'refs/heads/stable'
+      - name: Build a snapcraft configuration file
+        run: |
+          sh ci/snapcraft.sh
+        if: matrix.snap_arch != ''
+        env:
+          DO_SNAP: 1
+          SNAP_ARCH: ${{ matrix.snap_arch }}
+          SNAP_EDGE: 1 # skip-stable
+      - uses: jhenstridge/snapcraft-build-action@v1
```

Comment:
This is an individual developers snap action; I love James to bits, but if he leaves or gets hit by a bus this is going to break. Does Canonical not have a snap team account for this action to live under?

Reply:
Currently they don't.  The eventual goal for `rust-lang` itself is to mirror any actions we depend on which aren't github official ones or actions-rs I think.  For now I also trust James won't go bad, and once I know what the overall plan is, we can see about mirroring those actions or whatever is needed if Canonical don't do official actions in the snapcraft group or somesuch.

Reply:
Yeah, I have zero concerns about James going bad :) Could we at least ask if they are willing to make an actions repo in snapcraft/ ? Or has that been discussed and decided against?

Reply:
I believe James is seeking futher proof that his actions are good, before he persues officialdom.

Reply:
I have been discussing the work he did on https://forum.snapcraft.io/t/call-for-testing-github-action-for-snapcrtaft/14930 but as yet he's not indicated an intent to carry it forward further.

Reply:
/cc @jhenstridge

Reply:
I have brought up moving these actions to the `snapcore` github organisation, both to make it easier to share maintenance and to make updating them a bit structured.  These actions started as a holiday project to learn about this part of the GH Actions system, so it hasn't been a priority during the week.

You are right that there are security concerns about trusting an action that I solely control. The publish action in particular is in a prime position to silently exfiltrate the store credentials of any project that uses it.  The build action could similarly modify the project before building it: a little more involved and likely to be noticed in the logs, but still a possibility.

Actions provided by `snapcore` could also do all this, but as it is the same org that publishes Snapcraft, it is also in a position to do all of this.

I'll try and move forward the process of moving the projects over.